qoqo_qiskit_devices/tests/integration/device.rs

```diff
@@ -176,9 +176,9 @@ fn test_to_generic_device(device: IBMDevice, pyo3_device: Py<PyAny>) {
 fn test_initialization_from_identifier(name: &str, device: Py<PyAny>) {
     pyo3::prepare_freethreaded_python();
     Python::with_gil(|py| {
-        let device_type = device.as_ref(py).get_type();
+        let device_type = device.bind(py).get_type();
         let result = qoqo_qiskit_device_from_ibmq_identifier(name).unwrap();
-        let resulting_type = result.as_ref(py).get_type();
+        let resulting_type = result.bind(py).get_type();
```

Comment:
This seems fine, but I found that I could just remove the "as_ref" in many places